models/classes/runner/session/TestSession.php

```diff
@@ -189,24 +189,6 @@ public function getItemAttemptTag(RouteItem $routeItem)
         return $itemId . '#' . $occurrence . '-' . $itemSession['numAttempts']->getValue();
     }
 
-    /**
-     * Initializes the timer for the current item in the TestSession
-     *
-     * @param $timestamp
-     * @throws \oat\taoTests\models\runner\time\InvalidDataException
-     */
-    public function initItemTimer($timestamp = null)
```

Comment:
I see this method is overridden in `taoAct` extension, https://github.com/oat-sa/extension-tao-act/blob/master/model/runner/ActTestSession.php#L57, this change should address that.

Reply:
I got back this method for backward compatibility, I suggest to create a tech dept task.